test/__init__.py

```diff
@@ -129,6 +129,10 @@
 }
 KMIP_CREDS = {"endpoint": os.environ.get("FLE_KMIP_ENDPOINT", "localhost:5698")}
 
+os.environ[
+    "MONGOB_LOG_MAX_DOCUMENT_LENGTH"
+] = "2000"  # Ensure Evergreen metadata doesn't result in truncation
```

Comment:
Suggest using setdefault so that we can still set this manually. For example I recently found it useful to set MONGOB_LOG_MAX_DOCUMENT_LENGTH when running the perf tests.